### DISCUSSION TITLE
Option to change font size for multiple presets (or a toggle for ignoring preset font size)

### DISCUSSION THREAD

**Discussion by NemukaPrime** [monkeytypegame/monkeytype#4605] [Ideas]
Hi, I would like an option for changing the font size for all presets, or at least being able to select in which presets to apply the font size.

**Problem:** When I change monitors of different sizes and scaling values, changing the preferred font size for 10 presets can get tedious.

I would like something like a toggle to ignore font size in presets so the user can can just focus on what I think are some of the main concerns when changing presets, which for me are the language, time, and word count. 
Another option would be changing font size for all presets, or a subset of them to make it easier for someone who wants to change font size for a lot of presets. Thank you.

**Comment by Miodec** [comment]
#4532 should help with this.

Also, remember you can quickly change font size with esc > font size

**Reply by NemukaPrime** [reply]
Yes, partial presets would be great. I do change font size with esc but usually I switch presets every few tests so it gets tedious and I usually just end up leaving the font size at a less ideal setting.
